pkg/printers/sarif.go

```diff
@@ -0,0 +1,110 @@
+package printers
+
+import (
+	"encoding/json"
+	"io"
+
+	"github.com/golangci/golangci-lint/pkg/result"
+)
+
+const (
+	sarifVersion   = "2.1.0"
+	sarifSchemaURI = "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.4.json"
```

Comment:
Maybe we should use the newest available version?
```suggestion
	sarifSchemaURI = "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.6.json"
```